pcdsdevices/ccm.py

```diff
@@ -270,23 +287,15 @@ def alio_to_theta(alio, theta0, gr, gd):
     docstring, either to indicate your success or to increment the hours
     counter below.
 
+    Conversion function
+    theta_angle = f(x) = 2arctan * [(sqrt(x^2 + D^2 + 2Rx) - D)/(2R + x)]
+    Note that for x = −R, θ = 2 arctan(−R/D)
+
     total hours spent here: 2
     """
-
-    low = -1.0
-    high = 1.0
-    timeout = 1.0
-    start = time.time()
-    while time.time() - start < timeout:
-        theta_guess = (low+high)/2
-        alio_calc = theta_to_alio(theta_guess, theta0, gr, gd)
-        if np.isclose(alio, alio_calc):
-            break
-        elif alio_calc > alio:
-            high = theta_guess
-        else:
-            low = theta_guess
-    return theta_guess
+    return theta0 + 2 * np.arctan(
```

Comment:
took out the  `180 / n.pi` from the old function, because we are now returning radians instead of degrees.

old_func:
```
  return gTheta0 + 180 / n.pi * 2 * n.arctan(
        (n.sqrt(alio ** 2 + gD ** 2 + 2 * gR * alio) - gD) / (2 * gR + alio)
    )
```